### DISCUSSION TITLE
DOC: Deploy on Azure

### DISCUSSION THREAD

**Discussion by roldengarm** [getfider/fider#1073] [General]
Very good documentation on how to deploy to Azure! There are two things I would like to add:

- Fider doesn't seem compatible with Postgres Flexible Server (out of the box). The Azure Portal suggests to use Flexible Server instead of Single Server, which I did. But then the app fails on startup with: `pq: extension "pg_trgm" is not allow-listed for "azure_pg_admin" users in Azure Database for PostgreSQL`
[Looks like this extension is indeed not available](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/concepts-extensions#postgres-13-extensions) 
- Another app setting is required `HOST_DOMAIN` which is not in the documentation. After adding it, it works.

**Comment by goenning** [comment]
Did it work on Single Server? We might need to change the docs.

The HOST_DOMAIN is no longer needed on the latest version (not released yet), so that's why there is a mismatch! I'll likely release it this week to avoid further confusion.

**Comment by roldengarm** [comment]
@goenning  yes it worked on Single Server. All good now.

Thanks!
